compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp

```diff
@@ -172,9 +179,17 @@ std::unique_ptr<luci::Module> BisectionSolver::run(const std::string &module_pat
   switch (_algorithm)
   {
     case Algorithm::Auto:
-      int16_front =
-        error_at_input_is_larger_than_at_output(nodes_depth, 0.5f * (max_depth + min_depth));
-      break;
+    {
+      auto model = read_module(module_path);
+      if (!_quantizer->quantize(model.get(), "uint8", layer_params))
+      {
+        std::cerr << "ERROR: Failed to quantize model" << std::endl;
+        return nullptr;
+      }
+      int16_front = error_at_input_is_larger_than_at_output(nodes_depth, _visq_data_path,
```

Comment:
(optioanl) this method is too long. please make it shorter.

Reply:
> (optional) this method is too long. please make it shorter.

Ok.